rna-transcription.json

```diff
@@ -0,0 +1,105 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
```

Comment:
Actually, let's not use -1 for expected. If expected is a string, then let's use an empty string.